modules/ocf_kubernetes/manifests/master/secrets.pp

```diff
@@ -0,0 +1,18 @@
+class ocf_kubernetes::master::secrets {
+
+  file {
+    default:
+      require => Package['kubectl'];
+
+    '/etc/ocf-kubernetes/secrets':
+      ensure  => 'directory',
+      source  => 'puppet:///kubernetes-secrets',
```

Comment:
err... is this the right location of the file? if this is meant to point somewhere in the private share, you should use the `kubernetes` directory that we setup.